src/Symfony/Bundle/FrameworkBundle/EventListener/TwigTemplateListener.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\EventListener;
+
+use Symfony\Bundle\FrameworkBundle\Templating\TemplatedResponseInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+class TwigTemplateListener implements EventSubscriberInterface
```

Comment:
A Twig listener should be in TwigBundle or in the Twig bridge (if it depends on FramworkBundle, it should be in TwigBundle).
FrameworkBundle is not allowed to depend on Twig.